app/models/employer.rb

```diff
@@ -0,0 +1,15 @@
+class Employer < User
+  has_many :workers
+
+  def hire_worker(worker_id)
+    worker = Worker.find worker_id
+    worker.employer_id = id
+    worker.save!
```

Comment:
What do you intend to do when the hiring process becomes more complicated than just assigning an employer?